packages/lib/RotatingLogs.ts

```diff
@@ -0,0 +1,73 @@
+const moment = require('moment');
+const { FsDriverDummy } = require('./fs-driver-dummy.js');
+
+export default class RotatingLogs {
+
+	public static fsDriver_: any = null;
+
+	private logFileDir: string = '';
+	private logFileName: string = 'log.txt';
+	private logFileExpirationTimeInDays: number = 30;
+	private logFileMaximumSizeInBytes: number = 1024 * 1024 * 10;
+
+	public constructor(logFileDir: string) {
+		this.setLogFileDir(logFileDir);
+	}
+
+	public setLogFileDir(value: string) {
+		this.logFileDir = value;
+	}
```

Comment:
This could be private since you don't use anywhere else. In fact you don't need a method here, just set logFileDir in the constructor